### PR TITLE
Update OpenMP Training

### PR DESCRIPTION
**Pull request by strongwar**
# Existing Sample Changes
## Description

Updated OpenMP Training for C++ and Fortran, added new sections for Host Device Concurrency, and Setting Target Device for Debugging. No added code only added Markdown sections.

Fixes Issue# 

## External Dependencies

List any external dependencies created as a result of this change.
None

## Type of change

Please delete options that are not relevant. Add a 'X' to the one that is applicable. 

- [X] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [X] Command Line
- [ ] oneapi-cli
- [ ] Visual Studio
- [ ] Eclipse IDE
- [ ] VSCode
- [X] When compiling the compliler flag "-Wall -Wformat-security -Werror=format-security" was used
- [x] Run IPLDT

Ran Jupyter Notebooks on devcloud. Also compiled and ran code samples.

### CODE REVIEW

**Comment by JoeOster** [comment]
@strongwar please review and fix your conflicts

**Comment by JoeOster** [comment]
@strongwar This should have been 2 PRs as its 2 different subjects that I can identify so far. I am not sure I see the need for the new folder structures unless you have more sample changes coming 

Also for the new new DirectProgramming/Fortran/Jupyter/OpenMP_Offload_Fortran/OpenMP Welcome_Fortran.ipynb, you create a new folder for fortran and then call your sample...fortran, it becomes redundant.

@tomlenth do you have any feedback on the proposed folder structure from a UIx perspective

**Comment by strongwar** [comment]
Reverted guid back to original
fix folders back to original structure

**Review by JoeOster — CHANGES_REQUESTED** [review]

File: DirectProgramming/C++/Jupyter/OpenMP-offload-training/OpenMP Welcome_C++.ipynb [hunk 1]
```diff
@@ -0,0 +1,62 @@
+{
```
[the remark this answers is not in the document]
Comment (strongwar) [reply DirectProgramming/C++/Jupyter/OpenMP-offload-training/OpenMP Welcome_C++.ipynb]: Fixed the file names, back to original for C++ and Fortran

**Review by JoeOster — APPROVED** [review]

**Review by JoeOster — COMMENTED** [review]
This sample is missing some elements

No license file - for a MIT license, use https://github.com/oneapi-src/oneAPI-samples/blob/master/License.txt

**Comment by JoeOster** [comment]
@tomlenth - Sample should require a Readme for each sample and none of the samples in this Jupyter folder has one, is this a miss or did we decide to skip it? The readme at the root is very generic and doesn't specify the samples contained...

**Review by tomlenth — CHANGES_REQUESTED** [review]
We definitely don’t need a readme for each sample given that it’s a jupyter notebook.  That said, it would be ideal to follow the exact format as the general DPC++ notebook, including the description of each directory. In addition, it’d be really nice to use the same directory naming convention with numbers to show the order. And, please add in the license mention.

**Review by JoeOster — CHANGES_REQUESTED** [review]
Per tom - We definitely don’t need a readme for each sample given that it’s a jupyter notebook. That said, it would be ideal to follow the exact format as the general DPC++ notebook, including the description of each directory. In addition, it’d be really nice to use the same directory naming convention with numbers to show the order. And, please add in the license mention.

**Review by tomlenth — APPROVED** [review]
would prefer that the directories are number like the other juypter trainings but can live with this for now.  will be more important to number as the number of directories increases since the material is meant to be consumed in a particular sequence.

**Comment by strongwar** [comment]
Updated pdfs

**Review by JoeOster — APPROVED** [review]

**Pull request merged**